docs/source/guide/pec-1-intro.md

```diff
@@ -28,6 +28,11 @@ operation.
 
 +++
 
+PEC works in two main stages: generate noise-scaled circuits via inserting quasi-probability representations of noisy gates, and combining the resulting measurements post-execution.
+
+This workflow can be executed by a single call to {func}`.execute_with_pec`.
+If more control is needed over the protocol, Mitiq provides {func}`.generate_sampled_circuits` and {func}`.pec.combine_results` to handle the first and second steps respectively.
```

Comment:
Let's add a statement up here saying something like the following (but more formally).

> In section XYZ we'll apply the protocol in a single step, and in section ABC we'll show how you can apply the technique stepwise.